en/docs/learn/tibco-migration-assistant.md

```diff
@@ -0,0 +1,192 @@
+    <bpws:partnerLinks>
+        <bpws:partnerLink name="HttpReceiver" partnerLinkType="tns:HttpReceiver" myRole="use"/>
+        <bpws:partnerLink name="helloworld" partnerLinkType="tns:helloworld" myRole="use"/>
+    </bpws:partnerLinks>
+    <bpws:variables>
+        <bpws:variable name="get" messageType="tns:getRequest"/>
+    </bpws:variables>
+    <bpws:scope name="scope">
+        <bpws:flow name="flow">
+            <bpws:links/>
+            <bpws:pick createInstance="yes" name="pick">
+                <bpws:onMessage operation="get"
+                    partnerLink="helloworld"
+                    portType="ns0:helloworld"
+                    variable="get">
+                    <bpws:scope name="scope1">
+                        <bpws:flow name="flow1">
+                            <bpws:links/>
+                            <bpws:reply name="getOut" operation="get"
+                                partnerLink="helloworld"
+                                portType="ns0:helloworld">
+                                <tibex:inputBinding expressionLanguage="urn:oasis:names:tc:wsbpel:2.0:sublang:xslt1.0">&lt;?xml version="1.0" encoding="UTF-8"?&gt;&lt;xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:tns="http://xmlns.example.com" version="2.0"&gt;     &lt;xsl:template name="getOut-input" match="/"&gt;         &lt;helloworld&gt; &lt;response&gt; helloworld &lt;/response&gt;&lt;/helloworld&gt;     &lt;/xsl:template&gt; &lt;/xsl:stylesheet&gt;</tibex:inputBinding>
+                                <tibex:inputBindings>
+                                    <tibex:partBinding
+                                    expression="&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
+&lt;xsl:stylesheet xmlns:xsl=&quot;http://www.w3.org/1999/XSL/Transform&quot; xmlns:tns=&quot;http://xmlns.example.com&quot; version=&quot;2.0&quot;&gt;
+    &lt;xsl:template name=&quot;getOut-input&quot; match=&quot;/&quot;&gt;
+        &lt;response&gt; helloworld &lt;/response&gt;
+    &lt;/xsl:template&gt;
+&lt;/xsl:stylesheet&gt;" expressionLanguage="urn:oasis:names:tc:wsbpel:2.0:sublang:xslt1.0"/>
+                                </tibex:inputBindings>
+                            </bpws:reply>
+                        </bpws:flow>
+                    </bpws:scope>
+                </bpws:onMessage>
+            </bpws:pick>
+        </bpws:flow>
+    </bpws:scope>
+</bpws:process>
+```
+2. Execute `bal migrate-tibco <path to helloworld.bwp> -o converted`. This will create the `converted` directory and create a Ballerina package inside it.
+
+### Step 3
+1. Open VS Code inside the `converted` directory
+```bash
+$ code ./test
+```
+2. Click Ballerina Integrator button in the left side bar to open Ballerina package in Ballerina Integrator.
+
+## Output
+
+### Migration summary
+
+- When you run the tool it will log the number of activities it detected for each process along with the number of activities it failed to convert, if any.
```

Comment:
```suggestion
- When you run the tool, it will log the number of activities it detected for each process along with the number of activities it failed to convert, if any.
```